src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String model;
```

Comment:
Don't forget about access modifiers for your custom fields. It's a good practice to make fields `private` and provide public getters and setters for access, which you have done correctly here.